recipe/conda_build_config.yaml

```
@@ -16,3 +16,4 @@ rust_std_extra:
   - i686-linux-android
   - wasm32-unknown-unknown
   - x86_64-linux-android
+  - x86_64-pc-windows-msvc
```

Since this output is already made for windows. Build it only on linux.
```suggestion
  - x86_64-pc-windows-msvc   # [linux and x86_64]
```